lib/platform/gitlab/index.md

```diff
@@ -0,0 +1,7 @@
+# GitLab and GitLab Hosted
```

Comment:
```suggestion
# GitLab
```

Reply:
This was chosen to align with the platform documentation at https://docs.renovatebot.com/modules/platform/ (IE, documentation of the platform), and not the name of the module this is in (IE, documentation of the module). Happy to take this change on, but please check the titles on lib/platform/github/index.md and lib/platform/azure/index.md and see if you'd like them similarly renamed for consistency of documentation.

Reply:
@viceice, It's safe to assume that the other files are titled appropriately?

Reply:
Yes, please use my suggestion for now.